meta/autoid/autoid.go

```diff
@@ -319,13 +335,44 @@ func (alloc *allocator) Alloc(tableID int64, n uint64) (int64, int64, error) {
 	alloc.mu.Lock()
 	defer alloc.mu.Unlock()
 	if alloc.isUnsigned {
-		return alloc.alloc4Unsigned(tableID, n)
+		return alloc.alloc4Unsigned(tableID, n, increment, offset)
 	}
-	return alloc.alloc4Signed(tableID, n)
+	return alloc.alloc4Signed(tableID, n, increment, offset)
+}
+
+// CalcNeededIDs is exported for test.
+func CalcNeededIDs(base, n, increment, offset int64, isUnsigned bool) int64 {
```

Comment:
I think we should rename this function, `CalcNeededIDs` is not a good name.

Reply:
Addressed.